packages/edit-site/src/store/test/actions.js

```diff
@@ -78,13 +78,6 @@ describe( 'actions', () => {
 				actionName: 'setPage',
```

Comment:
This needs to check for `done: true`.

Reply:
I actually had it check for `done: true` but [removed](1a3ab30239e165808d96f7cd754b852d8a14a08e) it after unit tests were failing, and I couldn't quite work out why :grimacing: 

I'll try reverting that commit :thinking:

Reply:
[Failing again.](https://travis-ci.com/github/WordPress/gutenberg/jobs/352819756) I don't quite see why this isn't working, do you have any ideas?

Reply:
I un-reverted that commit. AFAICS, this is failing since we're `return`ing a `dispatch` action here, which is a generator itself.

Reply:
`dispatch` just returns an object. Something else is wrong here.

Reply:
I haven't managed to track this down, but I might slap a `// FIXME` comment on this for the time being since we've noticed that the `gutenberg_find_template_post_and_parts` call in `lib/edit-site-page.php` that we're removing here might account partly for #23662.

Reply:
`master`: 274 queries, 148,8 ms
`This branch`: 28 queries, 5.4 ms

:microphone: :droplet:

Reply:
Sounds good.

Reply:
That's a pretty nice performance improvement <3

Reply:
Not sure if I'm missing something, but could you elaborate on how you were able to arrive at those perf numbers Bernie?

I'm trying to access the debug bar locally with this branch checked out but can't seem to get the query dashboard to display. (I'm using query params like so `http://localhost:8888/wp-admin/admin.php?page=gutenberg-edit-site&debug-bar=1`)

Reply:
You need the debug bar plugin installed and a few config constants set. You can use a `.wp-env.override.json` in your local Gutenberg source checkout dir to achieve both. This is mine (also links `theme-experiments` themes).

```json
{
        "core": "../wordpress-develop/build",
        "plugins": [ ".", "https://downloads.wordpress.org/plugin/debug-bar.zip" ],
        "themes": [ "WordPress/theme-experiments" ],
        "config": { "WP_DEBUG": true, "SCRIPT_DEBUG": true, "SAVEQUERIES": true }
}
```